rasa/core/test.py

```diff
@@ -144,24 +137,23 @@ class WronglyClassifiedUserUtterance(UserUttered):
 
     type_name = "wrong_utterance"
 
-    def __init__(
-        self,
-        text,
-        correct_intent,
-        correct_entities,
-        parse_data=None,
-        timestamp=None,
-        input_channel=None,
-        predicted_intent=None,
-        predicted_entities=None,
-    ):
-        self.predicted_intent = predicted_intent
-        self.predicted_entities = predicted_entities
+    def __init__(self, event: UserUttered, eval_store: EvaluationStore):
+
+        if eval_store.intent_predictions == list():
```

Comment:
Are you checking if `eval_store.intent_predictions` is an empty list? Why not using `if not eval_store.intent_predictions` as we use it almost everywhere else?

Reply:
Yes, I thought this was a bit more verbal

Reply:
let's keep it consistent with the rest of the codebase and use `if not eval_store.intent_predictions`